### ISSUE TITLE
Integrate filestack on admin portal

### ISSUE THREAD

**Issue by viviancan** [Iridescent-CM/technovation-app#3661]

**Comment by viviancan** [comment]
Refs #3613

**Comment by viviancan** [comment]
This is on QA and working as expected

**Comment by dboyer** [comment]
Checked on QA and this is working for students but not for mentors. Not sure if the mentor part was coming in another ticket but wanted to log the behavior just in case:

1. Logged in as an admin
2. Clicked to edit a student participant and uploaded a new image. Saved changes.
3. Saw a banner that the top of the page indicating changes were saved
4. New photo visible when I logged in as that participant
5. Clicked to edit a mentor participant and uploaded a new image. Clicked to save changes
6. No banner appeared indicating changes were saved
7. When I returned to the admin page for that participant, the default hedgehog photo was still there, not the new photo
